modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -330,7 +371,7 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
     // Run the script
     for (auto &op : m_ops)
     {
-        // (5)
+        // (5), (6)
```

Comment:
What is this comment?

Reply:
Steps 5 and 6 of the above algorithm are there. It seems "Step 6" comment has been removed during some refactoring.